library/node

```diff
@@ -4,34 +4,34 @@ Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@n
 GitRepo: https://github.com/nodejs/docker-node.git
 GitFetch: refs/heads/main
 
-Tags: 19-alpine3.16, 19.6-alpine3.16, 19.6.0-alpine3.16, alpine3.16, current-alpine3.16
+Tags: 19-alpine3.16, 19.6-alpine3.16, 19.6.1-alpine3.16, alpine3.16, current-alpine3.16
 Architectures: amd64, arm32v6, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: f996e97a2e6d2aae2de3b869e083a253733f07a8
+GitCommit: e75fa5270326ffaff8fee03153f3bf16860084d4
 Directory: 19/alpine3.16
 
-Tags: 19-alpine, 19-alpine3.17, 19.6-alpine, 19.6-alpine3.17, 19.6.0-alpine, 19.6.0-alpine3.17, alpine, alpine3.17, current-alpine, current-alpine3.17
+Tags: 19-alpine, 19-alpine3.17, 19.6-alpine, 19.6-alpine3.17, 19.6.1-alpine, 19.6.1-alpine3.17, alpine, alpine3.17, current-alpine, current-alpine3.17
 Architectures: amd64, arm32v6, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: f996e97a2e6d2aae2de3b869e083a253733f07a8
+GitCommit: e75fa5270326ffaff8fee03153f3bf16860084d4
 Directory: 19/alpine3.17
 
-Tags: 19, 19-bullseye, 19.6, 19.6-bullseye, 19.6.0, 19.6.0-bullseye, bullseye, current, current-bullseye, latest
+Tags: 19, 19-bullseye, 19.6, 19.6-bullseye, 19.6.1, 19.6.1-bullseye, bullseye, current, current-bullseye, latest
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: f996e97a2e6d2aae2de3b869e083a253733f07a8
+GitCommit: e75fa5270326ffaff8fee03153f3bf16860084d4
 Directory: 19/bullseye
 
-Tags: 19-bullseye-slim, 19-slim, 19.6-bullseye-slim, 19.6-slim, 19.6.0-bullseye-slim, 19.6.0-slim, bullseye-slim, current-bullseye-slim, current-slim, slim
+Tags: 19-bullseye-slim, 19-slim, 19.6-bullseye-slim, 19.6-slim, 19.6.1-bullseye-slim, 19.6.1-slim, bullseye-slim, current-bullseye-slim, current-slim, slim
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: f996e97a2e6d2aae2de3b869e083a253733f07a8
+GitCommit: e75fa5270326ffaff8fee03153f3bf16860084d4
 Directory: 19/bullseye-slim
 
-Tags: 19-buster, 19.6-buster, 19.6.0-buster, buster, current-buster
+Tags: 19-buster, 19.6-buster, 19.6.1-buster, buster, current-buster
 Architectures: amd64, arm32v7, arm64v8
-GitCommit: f996e97a2e6d2aae2de3b869e083a253733f07a8
+GitCommit: e75fa5270326ffaff8fee03153f3bf16860084d4
 Directory: 19/buster
 
-Tags: 19-buster-slim, 19.6-buster-slim, 19.6.0-buster-slim, buster-slim, current-buster-slim
+Tags: 19-buster-slim, 19.6-buster-slim, 19.6.1-buster-slim, buster-slim, current-buster-slim
 Architectures: amd64, arm32v7, arm64v8
-GitCommit: f996e97a2e6d2aae2de3b869e083a253733f07a8
+GitCommit: e75fa5270326ffaff8fee03153f3bf16860084d4
 Directory: 19/buster-slim
 
 Tags: 18-alpine3.16, 18.14-alpine3.16, 18.14.0-alpine3.16, hydrogen-alpine3.16, lts-alpine3.16
```

Comment:
the v18 update is messed up, see https://github.com/nodejs/docker-node/issues/1843#issuecomment-1434271253

hopefully a fix incoming